webapp/components/path.js

```diff
@@ -78,6 +78,14 @@ const PathInfo = (superClass) => class extends superClass {
     return path && path === '/';
   }
 
+  isParentDir(path, dir) {
+    if (dir.startsWith(path)) {
+      const relativePath = dir.substring(path.length);
+      return relativePath.split('/').filter(p => p).length === 1;
```

Comment:
What's the point of `filter(p => p)`?

Reply:
e.g. '/abd/'.split('/')) returns ['', 'abd, ''].`filter(p =>p)` allows us to remove empty strings